address_book/cmd/client/main.go

```diff
@@ -4,13 +4,14 @@ import (
 	"addressbook/internal/pb"
 	"context"
 	"log"
+	"os"
 
 	"google.golang.org/grpc"
 )
 
 func main() {
 	ctx := context.Background()
-	conn, err := grpc.Dial(":8080", grpc.WithInsecure(), grpc.WithBlock())
+	conn, err := grpc.Dial(":"+os.Getenv("PORT"), grpc.WithInsecure(), grpc.WithBlock())
```

Comment:
please think how to use cli params to run the service

you can add it in the next PR

Reply:
ok